qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -559,15 +583,24 @@ def evaluate(self, x_vec: np.ndarray, y_vec: np.ndarray = None) -> np.ndarray:
                     statevectors.append(results.get_statevector(j))
 
             offset = 0 if is_symmetric else len(x_vec)
-            matrix_elements = [
-                self._compute_overlap(idx, statevectors, is_statevector_sim, measurement_basis)
-                for idx in list(zip(mus, nus + offset))
-            ]
+            for (
+                i,
+                j,
+            ) in zip(mus, nus):
```

Comment:
This is really how black wants this or is it an aspect of the trailing , after j that is causing this formatting. If I compare to something below (now deleted), which has more in it, that went on one line ok i.e. (line 567 below deleted)
>for i, j, value in zip(mus, nus, matrix_elements):

Reply:
This was a `black` artifact from the previous version. Now it is just one line `for (i, j) in zip(mus, nus):`. Thank you.